lgpl.txt

```diff
@@ -0,0 +1,502 @@
+                  GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
hmm .. not all projects are under lgpl, so maybe we shouldn't add it for the org?